packages/c/cantarell-fonts/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : cantarell-fonts
 version    : '0.303.1'
-release    : 11
+release    : 12
 source     :
     - https://download.gnome.org/sources/cantarell-fonts/0.303/cantarell-fonts-0.303.1.tar.xz : f9463a0659c63e57e381fdd753cf1929225395c5b49135989424761830530411
+homepage   : https://cantarell.gnome.org/
 license    : OFL-1.1
 component  : desktop.gnome.core
 homepage   : https://gitlab.gnome.org/GNOME/cantarell-fonts
```

Comment:
There are two homepage keys in `package.yml` the newer one is not updated in `pspec_x86_64.xml` file.